src/java.desktop/windows/native/libawt/windows/awt_Component.cpp

```diff
@@ -3362,10 +3362,10 @@ BOOL AwtComponent::IsNumPadKey(UINT vkey, BOOL extended)
     return FALSE;
 }
 static void
-resetKbdState( BYTE kstate[256]) {
+resetKbdState( BYTE (&kstate)[256]) {
     BYTE tmpState[256];
```

Comment:
> Will this resolve Phil's concern? Both arrays will use the same size.

@prrace Does this address your concerns? If it does I will commit it

Reply:
That's fine since that  what I was asking for in my comment a week ago
> Why can't we EVERYWHERE use the definition used to create the array passed in KB_STATE_SIZE = 256 instead of a mixture.